docs/command-line-interface/ds-load-cli/transform.mdx

```diff
@@ -0,0 +1,257 @@
+
+    {{ if and ($.phone_number) (ne $.phone_number "") ($.phone_verified) }}
+    ,
+      {
+          "key": "{{ $.phone_number }}",
+          "type": "identity",
+          "properties": {
+            "kind": "IDENTITY_KIND_PHONE",
+            "provider": "auth0",
+            {{ fromEnv "connection_id" "ASERTO_CONNECTION_ID" }},
+            "verified": {{ .phone_verified }}
+          }
+      }
+    {{ end }}
+```
+
+---
+If Roles were returned from the Auth0 plugin, they are transformed into objects of type `group` with a key consisting of the role name.
+
+```go
+    {{ if $.roles }}, {{ end }}
+
+    {{ range $i, $element := $.roles }}
+      {{ if $i }},{{ end }}
+      {
+        "key": "{{$element.name}}",
+        "type": "group",
+        "properties":{
+          {{ fromEnv "connection_id" "ASERTO_CONNECTION_ID" }}
+        }
+      }
+    {{ end }}
+  ],
+```
+
+---
+Next, the transform generates a set of relations, to link the objects (user, identity, group) with each other.
+
+The first two relations link the `identity` and `user` objects through the `identifier` relationship.
+
+The next two relations are generated for the `username` and `phone_number` identities, if those were generated as objects.
+
+```go
+  "relations":[
+      {
+      "relation": "identifier",
+              "subject": {
+                  "type": "user",
+                  "key": "{{ $.email }}"
+              },
+              "object": {
+                  "type": "identity",
+                  "key": "{{ $.user_id }}"
+              }
+      },
+      {
+      "relation": "identifier",
+              "subject": {
+                  "type": "user",
+                  "key": "{{ $.email }}"
+              },
+              "object": {
+                  "type": "identity",
+                  "key": "{{ $.email }}"
+              }
+      }
+
+    {{ if and ($.username) (ne $.username "") }}
+    ,
+      {
+      "relation": "identifier",
+              "subject": {
+                  "type": "user",
+                  "key": "{{ $.email }}"
+              },
+              "object": {
+                  "type": "identity",
+                  "key": "{{ $.username }}"
+              }
+      }
+    {{ end }}
+
+    {{ if and ($.phone_number) (ne $.phone_number "") ($.phone_verified) }}
+    ,
+      {
+      "relation": "identifier",
+              "subject": {
+                  "type": "user",
+                  "key": "{{ $.email }}"
+              },
+              "object": {
+                  "type": "identity",
+                  "key": "{{ $.phone_number }}"
+              }
+      }
+    {{ end }}
+```
+
+---
+Finally, if roles were retrieved for each user (and a group generated for each of the roles) the transform generates a `member` relation between the user and group.
+
+```go
+
+    {{ if $.roles }}, {{ end }}
+
+    {{ range $i, $element := $.roles }}
+      {{ if $i }},{{ end }}
+      {
+      "relation": "identifier",
```

Comment:
You were right in https://asertoworkspace.slack.com/archives/C01HCE8B9NU/p1693357842068199 and the relation type should be `member` not `identifier`